app/controllers/subscriptions_controller.rb

```diff
@@ -124,12 +124,12 @@ def email
 
   def search_criteria_params
     params.require(:search_criteria)
-          .permit(:keyword, :location, :organisation_slug, :radius, job_roles: [], ect_statuses: [], subjects: [], phases: [], working_patterns: [])
+          .permit(:keyword, :location, :organisation_slug, :radius, teaching_job_roles: [], teaching_support_job_roles: [], non_teaching_support_job_roles: [], ect_statuses: [], subjects: [], phases: [], working_patterns: [])
   end
 
   def subscription_params
     params.require(:jobseekers_subscription_form)
-          .permit(:email, :frequency, :keyword, :location, :organisation_slug, :radius, job_roles: [], ect_statuses: [], subjects: [], phases: [], working_patterns: [])
+          .permit(:email, :frequency, :keyword, :location, :organisation_slug, :radius, teaching_job_roles: [], ect_statuses: [], subjects: [], phases: [], working_patterns: [])
```

Comment:
Shouldn't `teaching_support_job_roles`  and  `non_teaching_support_job_roles` also be added here?